drivers/hmis/app/graphql/mutations/submit_form.rb

```diff
@@ -117,10 +117,13 @@ def resolve(input:)
     end
 
     private def perform_side_effects(record)
-      if record.is_a?(Hmis::Hud::Client)
-        if record.new_record?
+      # If a Client record was created or updated, queue up IdentifyDuplicates job. This creates the warehouse destination client.
```

Comment:
Does this get called when someone gets a service as well?  If it does, we should add:
```
enrollment.invalidate_processing!
GrdaWarehouse::Tasks::ServiceHistory::Enrollment.delay.batch_process_unprocessed!
```

Reply:
might need to move the `invalidate_processing!` method into a concern, I think that's just on `GrdaWarehouse::Hud::Enrollment`

Reply:
Ok I'm guessing we only need to do that for HUD services, not custom services?

Reply:
https://github.com/greenriver/hmis-warehouse/pull/3519/commits/f193a95927f363db7defa4f47c8d9c0b0648d33d

Reply:
🧐 okay I'll add for that too

When about when services or cls are deleted, do we need to do this?

Reply:
Yes!  Any change, including changes to the date should trigger this.

Reply:
Actually, lets move this conversation to https://www.pivotaltracker.com/story/show/186062176. A bit out of scope for this PR and I think there are other places (like ENrollment creation) where this needs to happen